### DISCUSSION TITLE
Audio Gridder losing sync between servers

### DISCUSSION THREAD

**Discussion by GreenNeedleRecords** [apohl79/audiogridder#874] [General]
Hi, I am using AG 1.11 on 3 windows 10 machines, one main Audio DAW and 2 servers. I have a project going now where the two servers keep going out of sync timing wise with the main DAW. Any ideas how to fix this?

**Comment by JamesXXV** [comment]
you may want to update all AG installations to a more recent version like 1.2.0 beta 4-6
